octoprint_octorelay/__init__.py

```diff
@@ -363,7 +370,7 @@ def print_stopped(self):
     def turn_off_pin(self, relay_pin, inverted, cmdOFF):
         GPIO.setup(relay_pin, GPIO.OUT)
         # XOR with inverted
-        GPIO.output(relay_pin, inverted != False)
+        GPIO.output(relay_pin, inverted is not False)
```

Comment:
regarding this.
this syntax is really better since it's strict, not type casting.
However, neither the method `turn_off_pin` nor its caller do ensure the type of `inverted` parameter as boolean.
Meaning that we actually need a more loose comparison here until the type will be guaranteed.

Reply:
```suggestion
        GPIO.output(relay_pin, inverted != False)
```